lib/plugin/autoDelay.js

```diff
@@ -3,7 +3,7 @@ const store = require('../store');
 const recorder = require('../recorder');
 const event = require('../event');
 const log = require('../output').log;
-const supportedHelpers = require('./standardActingHelpers');
+const supportedHelpers = JSON.parse(JSON.stringify(require('./standardActingHelpers')));
```

Comment:
```suggestion
const supportedHelpers = require('./standardActingHelpers').slice();
```

how about using `.slice()` cause it doesn’t change the original array. :)

Reply:
Oh! @PeterNgTr - More clean, now, I didn´t know this ! 😅